tests/test-multi-pow.c

```diff
@@ -0,0 +1,108 @@
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "99999999999999999A9RGRKVGWMWMKOLVMDFWJUHNUNYWZTJADGGPZGXNLERLXYWJE9WQH"
+        "WWBMCPZMVVMJUMWWBLZLNMLDCGDJ999999999999999999999999999999999999999999"
+        "999999999999YGYQIVD99999999999999999999TXEFLKNPJRBYZPORHZU9CEMFIFVVQBU"
+        "STDGSJCZMBTZCDTTJVUFPTCCVHHORPMGCURKTH9VGJIXUQJVHK99999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999999999999999999999999999999999999999999999999999999999999"
+        "9999999999999";
+
+    int mwm = 9;
```

Comment:
This should be changed to 14.
9 is for the testnet and 14 is for the mainnet.

You might copy for other test case.
However, I do not know why the `test-dcurl.c` uses 9 instead of 14.